SO_properties.py

```diff
@@ -2234,10 +2243,14 @@ def calculate_concentration(self, r):
 
     @lazy_property
     def concentration(self):
+        if self.skip_concentration:
```

Comment:
Following some discussion with Joop & Matthieu today, we have decided to skip calculating concentration for certain SO definitions.